### DISCUSSION TITLE
position property

### DISCUSSION THREAD

**Discussion by vikas-growisto** [fastn-stack/fastn#589] [Q&A]
```
-- ftd.column:
height.fixed.px: 400
width.fixed.px: 400
background.solid: yellow

-- ftd.column:
height.fixed.px: 200
width.fixed.px: 200
position: center
background.solid: red
anchor: parent

-- end: ftd.column

-- end: ftd.column
```
the position is not working can you plz give an example with different values.

**Comment by Arpita-Jaiswal** [comment]
We don't support `position`. Instead, we use `align-self`. Checkout ftd [cheatsheet](https://github.com/ftd-lang/ftd/blob/main/Cheatsheet.md).
